docs/utils.rst

```diff
@@ -137,11 +137,9 @@ Evaluation Utils
 Genomic Utilities
 -----------------
 
-.. autofunction:: deepchem.utils.genomics.seq_one_hot_encode
+.. autofunction:: deepchem.utils.genomics_utils.seq_one_hot_encode
 
-.. autofunction:: deepchem.utils.genomics.encode_fasta_sequence
```

Comment:
Could you add in
```
.. autofunction:: deepchem.utils.genomic_utils.encode_fasta_sequence
```

Reply:
I think the `encode_fasta_sequence ` is alomost same API as `encode_bio_sequence`. So, I removed it. For the backward compatibility, I export `encode_fasta_sequence` from `deepchem/utils/save.py`

If you think it is better to export  `encode_fasta_sequence `, I will fix this point 🙆‍♂️

Reply:
Ah that makes sense! Should be all good then :)